stpa_documentation_tool.appdata.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It would be nice to have appdata/desktop files contributed upstream and installed from CMake.